src/lib.rs

```diff
@@ -212,36 +275,38 @@ mod test {
             bool,
             usize, isize, char,
             (),
+            [u8; 4],
+            &[u8],
         );
     }
 
     #[test]
     fn ref_type() {
-        host_library!(&NonImplicitCloneType);
-        // `host_library!(NonImplicitCloneType)` doesn't compile
```

Comment:
@kirillsemyonkin static-assertions actually has a macro to test that a type does not implement something. So we can actually test those cases

Reply:
It might be a good idea; won't hurt at least